web/beacon-app/src/components/ui/CancelModal/CancelAcctModal.tsx

```diff
@@ -0,0 +1,30 @@
+import { memo, useState } from "react"
+import { Fragment } from "react"
+
+import { Modal } from "@rotational/beacon-core"
+
+import { Close } from "@/components/icons/close"
+
+function CancelAcctModal() {
+  const [isOpen, setIsOpen] = useState(true)
+
+  const closeModal = () => setIsOpen(false)
+
+    return(
+    <div>
+      <Modal
+        title="Cancel Account"
+        open={isOpen}
+        className="max-w-md"
```

Comment:
Setting the prop size to small didn't change the modal's width to make it look more like it does in the Figma design, so that is why a max width is declared here.